targets/UnrealMarketplacePlugin/examplesource/ExampleProject/BuildAndRunTestsIOS.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+usage="./BuildAndRunTestsIOS.sh <target ipa directory path> <path to project>"
+
+archivePath=$1
+projectPath=$2
+
+#verify inputs
+if [ "$#" -ne 2 ]; then 
+    echo $usage
+    exit 1
+fi
+
+#be safe and clean up old archives
+rm -fdr $archivePath/IOS
+
+#inject the testTitleData into the project prior to build
+cp $PF_TEST_TITLE_DATA_JSON $projectPath/Content/TestTitleData
+
+#build the archive
+./BuildIOS.sh $archivePath $projectPath
+buildResult=$?
+
+#remove the testTitleData from the project now that the build is finished
+rm -f $projectPath/Content/TestTitleData/testTitleData.json
+
+#exit if the build failed
+if [ $buildResult -ne 0 ]; then 
+    echo "BUILD FAILED!"
+    exit 1
+fi
+
+cd $archivePath/IOS
+
+#remove a preexisting payload folder from a prior unqip if it exists (it really shouldn't tho)
```

Comment:
>unqip [](start = 50, length = 5)

is this "unzip"?

Reply:
> > unqip [](start = 50, length = 5)
> 
> is this "unzip"?

Not sure what you're asking.  We do call unzip on 38.

Reply:
"remove a preexisting payload folder from a prior unqip" <- that :)

Reply:
Ohh hah, yes. I'll address it!